bin/dev/create_helm_charts_pr.sh

```diff
@@ -53,6 +57,25 @@ rm -rf stable/uaa
 cp -r ../bundle/helm/cf stable/
 cp -r ../bundle/helm/uaa stable/
 
+# Show the places in the charts having RC references
+grep -rn '.-rc' stable/{cf,uaa}
+
+# Fixing RC references.
+
+# CLUSTER_BUILD: "2.20.2-rc1"
+sed -i 's|-rc[0-9][0-9]*||' stable/cf/values.yaml
+sed -i 's|-rc[0-9][0-9]*||' stable/uaa/values.yaml
+
+# scfVersion: 2.20.2-rc1+cf12.17.0.0.g5c8dc458
+sed -i 's|-rc[0-9][0-9]*||' stable/cf/Chart.yaml
+sed -i 's|-rc[0-9][0-9]*||' stable/uaa/Chart.yaml
+
+# version: 2.20.2-rc1
+sed -i 's|-rc[0-9][0-9]*||' stable/cf/Chart.yaml
+sed -i 's|-rc[0-9][0-9]*||' stable/uaa/Chart.yaml
```

Comment:
There's no need to run the replace twice in the same file (even though there are two instances of it); `sed` will just do both, right?

Also, I still don't see it in _UAA_ `values.yaml`, just the CF copy.

```
$ grep -riP -- '-rc[^\s]' .
./helm/uaa/Chart.yaml:version: 2.20.3-rc1
./helm/uaa/Chart.yaml:scfVersion: 2.20.3-rc1+cf12.17.0.0.g7175b9de
./helm/cf/values.yaml:  CLUSTER_BUILD: "2.20.3-rc1"
./helm/cf/Chart.yaml:version: 2.20.3-rc1
./helm/cf/Chart.yaml:scfVersion: 2.20.3-rc1+cf12.17.0.0.g7175b9de
```

Reply:
Actually, right. With the modified pattern the first per file will catch them all. Oops.

Reply:
Addressed in commit 3266af72113342eac79ac2a55ba89260e0758ffc